src/webui/www/private/scripts/client.js

```diff
@@ -781,8 +782,38 @@ window.addEvent('load', function() {
         addMainWindowTabsEventListener();
         addSearchPanel();
     }
+
+    registerMagnetHandler();
 });
 
+function registerMagnetHandler() {
+    if (typeof navigator.registerProtocolHandler !== 'function') {
```

Comment:
curly brackets can be omitted here (single line below) and in other similar instances.